examples/main/README.md

```diff
@@ -17,60 +17,68 @@ This example program allows you to use various LLaMA language models in an easy
 
 To get started right away, run the following command, making sure to use the correct path for the model you have:
 
-#### Unix-based systems (Linux, macOS, etc.):
+### Unix-based systems (Linux, macOS, etc.):
+
+##### Input prompt (One-and-done)
 
 ```bash
-./llama-cli -m models/7B/ggml-model.bin --prompt "Once upon a time"
+./llama-cli -m models/gemma-1.1-7b-it.Q4_K_M.gguf --prompt "Once upon a time"
 ```
+##### Conversation mode (Allow for continuous interaction with the model)
 
-#### Windows:
-
-```powershell
-llama-cli.exe -m models\7B\ggml-model.bin --prompt "Once upon a time"
+```bash
+./llama-cli -m models/gemma-1.1-7b-it.Q4_K_M.gguf -cnv --ctx_size 0 --chat-template llama3
 ```
-
-For an interactive experience, try this command:
-
-#### Unix-based systems (Linux, macOS, etc.):
+##### For an interactive experience, try this command:
 
 ```bash
-./llama-cli -m models/7B/ggml-model.bin -n -1 --color -r "User:" --in-prefix " " -i -p \
+./llama-cli -m models\gemma-1.1-7b-it.Q4_K_M.gguf -n -1 --color -r "User:" --in-prefix " " -i -p \
 'User: Hi
 AI: Hello. I am an AI chatbot. Would you like to talk?
 User: Sure!
 AI: What would you like to talk about?
 User:'
 ```
+##### Infinite text from a starting prompt (you can use `Ctrl-C` to stop it):
+```bash
+./llama-cli -m models\gemma-1.1-7b-it.Q4_K_M.gguf --ignore-eos -n -1
+```
 
-#### Windows:
+### Windows:
```

Comment:
I don't think windows should be separated since it's just the same with linux/macos (except for the binary name)

This section should be refactored into just "How to run"